Remotion/ObjectBinding/Web.Test/IndividualControlTests/BocListUserControl.ascx.cs

```diff
@@ -50,7 +50,7 @@ public class BocListUserControl : BaseUserControl
   protected Button ChildrenListSetPageButton;
   protected Button ChildrenListAddRowButton;
   protected Button ChildrenListRemoveRowsButton;
-  protected Button ChildrenAddValidationErrorsButton;
+  protected CheckBox EnableValidationErrorsCheckbox;
```

Comment:
this change is an x-over with one of @Frazerus changesets? Changes look okay, btw.

Reply:
>  if there was a reason for storing the boolean in the session instead of using the checkbox directly. 
I have also mentioned this in my review with the recommendation to use just the checkbox, not the session.